web/docs/setup-guides/deploy-a-node.md

```diff
@@ -61,6 +61,15 @@ Download the latest [Debian package][tenzir-debian-package] and install it via
 dpkg -i tenzir-static-amd64-linux.deb
 ```
 
+If you want to remove tenzir from your Debian based Linux you can do so with
+```bash
+apt-get remove tenzir
+```
+If you want to also remove the state and leave no trace, you can use
+```bash
+apt-get purge tenzir
+```
```

Comment:
You can uninstall the Tenzir package as follows:

```bash
apt-get remove tenzir
```

Use `purge` instead of `remove` if you also want to delete the state directory and leave no trace:

```bash
apt-get purge tenzir
```

I couldn't use the ```suggestion mode due to the contained code blocks.